presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7016,6 +7017,42 @@ public void testCaseToMapOptimization()
     @Test
     public void testLikePrefix()
     {
-        assertQuery("select x like 'abc%' from (values 'abc', 'def', 'bcd') T(x)");
+        assertNotEquals(computeActual("EXPLAIN(TYPE DISTRIBUTED) select x like 'abc%' from (values 'abc', 'def', 'bcd') T(x)").getOnlyValue().toString().toLowerCase(Locale.ROOT).indexOf("substr"), -1);
+    }
+
+    @Test
+    public void testArraySort()
+    {
+        assertQuery("select array_sort(cast(array[5,6,4,null,3,7] as array<integer>))", "select array[3, 4, 5, 6, 7, null]");
+        assertQuery("select array_sort(array[5,6,4,null,3,7])", "select array[3, 4, 5, 6, 7, null]");
+
+        MaterializedResult result = computeActual("select array_sort(array[5,6,4,null,3,7, cast(9223372036854775807 as bigint)])");
+        assertSorted(result.getMaterializedRows().get(0).getFields());
+
+        result = computeActual("select array_sort(array[5.4,6.1,4.9,null,4.5,7.8,5.2])");
```

Comment:
Can we test that the result is indeed a sorted representation of the original: ie same size and set of values. Also can we test a case of all null values and length less than 2.

Reply:
we can't do that for non-integer types

Reply:
there were other tests for this function before - we just specialized it. So no need to add other tests.